src/Psl/ObjectOriented/class_exists.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Psl\ObjectOriented;
+
+use function class_exists as php_class_exists;
+
+/**
+ * Checks if the class has been defined.
+ *
+ * Example:
+ *
+ *      ObjectOriented\class_exists('Psl\Collection\Vector');
+ *      => Bool(true)
+ *
+ *      ObjectOriented\class_exists('Psl\Collection\Foo');
+ *      => Bool(false)
+ *
+ * @psalm-assert-if-true class-string $class_name
+ */
+function class_exists(string $class_name): bool
```

Comment:
How about adding a `class_is_loaded()` function, which does a `class_exists`, but with the 2nd param being false?

Reply:
yea, probably a good idea, not sure about the name `class_included`? `class_loaded`?

Reply:
I'dd prefer `class_loaded`, since included is a bit strange in php context.